### PR TITLE
Add editMember section of UG

### PR DESCRIPTION
**Pull request by ph-nathan**
Closes #

What have you done in this PR?
* Update editMember section of UG

Dev Testing Steps / Screenshots

### CODE REVIEW

**Review by SelwynAng — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -84,6 +84,19 @@ Examples:
...
 
+### Edit a Member: `editMember`
+Edits the member at the specified index with the specified attributes.
+
+Format: `editMember MEMBER_INDEX [n/MEMBER_NAME] [g/GENDER] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]`
+
+* The index refers to the index number shown in the displayed member list.
+* The index **must be a positive integer** that is within the range of the length of the member list.
+* At least one field to edit must be provided.
+
```
Comment (dylkaw) [inline docs/UserGuide.md:95]: Should the constraints for each field be specified?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -84,6 +84,19 @@ Examples:
...
 * `deleteMember 10 ` deletes the 10th member in the member list.
 
+### Edit a Member: `editMember`
+Edits the member at the specified index with the specified attributes.
+
+Format: `editMember MEMBER_INDEX [n/MEMBER_NAME] [g/GENDER] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]`
+
+* The index refers to the index number shown in the displayed member list.
+* The index **must be a positive integer** that is within the range of the length of the member list.
+* At least one field to edit must be provided.
```
Comment (dylkaw) [inline docs/UserGuide.md:94]: Should the constraints for each field be specified?

**Review by dylkaw — CHANGES_REQUESTED** [review]
Just a comment on the constraints for each field.

File: docs/UserGuide.md (hunk 2 above)
Comment (ph-nathan) [reply docs/UserGuide.md:94]: Noted! Have included the constraints for mistake-prone fields like gender and email.

**Review by dylkaw — APPROVED** [review]

**Pull request merged**